tests/unit/forms.test.ts

```diff
@@ -85,6 +86,7 @@ test('deleteForm removes the correct uid form ', async () => {
 })
 
 test('create form has the correct path and method ', async () => {
+  // @ts-ignore
```

Comment:
Is this line still applicable?

Reply:
Is it necessary, no... But it prevents a red squiggly line as it's testing an invalid argument... 😅